webpack.config.js

```diff
@@ -1,4 +1,5 @@
 const path = require('path');
+const webpack = require('webpack');
```

Comment:
Why do we need to import webpack? I'm not sure it's needed for the config file — it's not being used anywhere?

Reply:
Sorry it's a fragment, I had to re-delete the webpack ESM export that I had experimented with.